CHANGELOG.md

```diff
@@ -37,6 +42,8 @@ and reindex everything.
   `index` and `deindex` methods
 - endpoints API changed
 - by default, documents are now stored in a separate Redis database
+- the key "id" has been removed from the geojson Feature root (but it's still
```

Comment:
`the key "id" is not required anymore in the loaded data and as such has been removed from the geojson Feature root`